### DISCUSSION TITLE
Noob Level: I'm overwhelmed by alerts

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#2935] [Unsupported Versions]
I have been using SO for about a year. I am now on SO 2. We have a really small network, more of a lab environment. Up to now I've just been using it for network monitoring. But I've recently installed the wazuh agent on just one desktop machine. And I'm getting a lot of alerts. Can someone point me to a tutorial or blog for some advice on how to manage this? I would like to install the agents on all the other machines.

**Comment by innovate-support** [comment]
I have the same problems filtering alerts.  I know it's possible, but the instructions are too vague for those of us not experts in this stuff.  I'm attempting to pay for support to accomplish my filtering needs.  Once I figure it out I'll try to post my results in these Discussions for others to use.

**Reply by ghost** [reply]
Thank you. There are some videos on their YouTube channel, but they're rather general (I'm not complaining, Mods). FWIW, I and my partner got our CompTIA certs (CySA+ and Security+) about a year ago and we are trying to develop low-cost cybersecurity solutions for small businesses and SO is perfect. We really need to accelerate our learnings on this. Anything you can share would be most welcome.

**Reply by Jesse-Cail** [reply]
I used SO 16 in the past and was familiar with editing the rules, but have been away for a while. I'm having a difficult time figuring out exactly where I should be editing for suppression. I've read about doing it in the global pillar using the suppressions under thresholds, but what I can't find is an exact how to do that... just vi the global.sls? A video on the YT Channel for creating suppressions would be awesome!

**Reply by BustedSec** [reply]
bump

**Reply by ghost** [reply]
Not sure what "bump" means...

**Reply by innovate-support** [reply]
I'm also trying to get suppression figured out here
https://github.com/Security-Onion-Solutions/securityonion/discussions/2910

Have discovered multiple issues

**Reply by BustedSec** [reply]
> Not sure what "bump" means...

https://www.dictionary.com/browse/bump#:~:text=Bump%20is%20an%20online%20slang,or%20thread's%20status%20and%20visibility.
